scripts/docker.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# Build (and optionally upload to Dockerhub) for CLI $1 & version $2.
+# additionally --release, -r to release container to Dockerhub
+# note: if you want to release as well, you should be logged in with `docker`.
+
+set -e
+
+DIR=`dirname $0`
+ROOT=$DIR/..
+
+CLI=$1
+VSN=$2
+RELEASE=$3
+
+if [ -z "$CLI" ]; then
+    echo "require cli as first parameter" 2>&1
+    exit 1
+fi
+
+if [ -z "$VSN" ]; then
+    echo "require version as second parameter" 2>&1
+    exit 1
+fi
+
+
+# build
+# disable CGO to make static binary
+CGO_ENABLED=0 go build                                                        \
+    -o $ROOT/${CLI}/build/$CLI                                                \
+    $ROOT/${CLI}
+
+# docker container
+sudo docker build --tag ipinfo/$CLI:$VSN $ROOT/$CLI/
+
+if [ "$RELEASE" = "-r" ] || [ "$RELEASE" = "--release" ]; then
+    # push on docker hub
+    sudo docker push ipinfo/$CLI:$VSN
+
+    # cleanup 
+    rm -rf $ROOT/$CLI/build
```

Comment:
Few things:

1. This is needed regardless of whether we release or not.
2. We can do it even before we push, i.e. right after `docker build` finishes, this is safe to run.
3. The `-f` is not needed.